libs/application/templates/driving-school-confirmation/src/fields/NationalIdWithName/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { FC, useEffect, useState } from 'react'
+import { Box, GridRow, GridColumn } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import { m } from '../../lib/messages'
+import { FieldBaseProps, getErrorViaPath } from '@island.is/application/core'
+import { gql, useLazyQuery } from '@apollo/client'
+import { IdentityInput, Query } from '@island.is/api/schema'
+import { InputController } from '@island.is/shared/form-fields'
+import { useFormContext } from 'react-hook-form'
+import * as kennitala from 'kennitala'
+import { Student } from '../../types'
+
+const IdentityQuery = gql`
+  query IdentityQuery($input: IdentityInput!) {
+    identity(input: $input) {
+      name
+      nationalId
+    }
+  }
+`
+
+const NationalIdWithName: FC<FieldBaseProps> = ({ field, application }) => {
```

Comment:
im wondering why these changes are in here? this is something thats been recently merged to main in another PR